### ISSUE TITLE
[Feature Request]: Add option to set MARKER_START and MARKER_START_SKIP.

### ISSUE THREAD

**Issue by liuyib** [syavorsky/comment-parser#97]
Hello everyone. First of all, thank you for developing this awesome project.
But now, I have a request to parse the comments, as shown below:

```js
/*
 * @lc app=leetcode.cn id=1 lang=javascript
 *
 * https://leetcode-cn.com/problems/two-sum/description/
 *
 */
```

As you can see, this comment starts with `/*`, but comment-parser can only parse comments starting with `/**`. So I want to known whether it is possible to add a configuration item and let the user decide `MARKER_START` and `MARKER_START_SKIP`. https://github.com/syavorsky/comment-parser/blob/master/parser.js#L6-L7 

If possible, this will help me a lot.

**Comment by brettz9** [comment]
While this would not technically be jsdoc, there are some environments where jsdoc-style tags are allowed, e.g., multiline comments without double asterisks: https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html#ts-check , so I can see how this might be of potentially wider interest . (Although we currently use regular expressions for such parsing, our [no-bad-blocks](https://github.com/gajus/eslint-plugin-jsdoc#no-bad-blocks) rule in `eslint-plugin-jsdoc` tries to prevent blocks without double asterisks unless certain tags as these TypeScript tags are present.)

**Comment by syavorsky** [comment]
This is reasonable suggestion, I will try to make it configurable in upcoming 1.0

**Comment by liuyib** [comment]
Yes, I also considered that my requirements might affect the previous functionality of the commenter-parser, but if implemented, this would make the commenter-parser more flexible and usable.


 
---Original---

**Comment by liuyib** [comment]
Thank you very much. It would be great if this could be done. ❤️



 
---Original---
